docs/development.md

```diff
@@ -117,7 +117,7 @@ Note: The windows build for Ollama is still under development.
 
 Install required tools:
 
-- MSVC toolchain - C/C++ and cmake as minimal requirements
+- MSVC 2019 Community Edition with C/C++ tools - You must build from a "Developer Shell" with the environment variables set
```

Comment:
Many people might already have 2022 – we can probably get away with saying to install MSVC regularly and for builds we use older stuff

Reply:
I'll need to test with msvc 2022 to make sure there aren't any path assumptions that break.

Reply:
Tested and it works.  (2022 + CUDA v12)  I've softened this test to just ensure an MSVC developer shell is detected regardless of version.